src/App.jsx

```diff
@@ -1,11 +1,25 @@
-function App() {
+import { Routes, Route } from "react-router-dom";
+import NavBar from "./components/NavBar";
+import Home from "./routes/Home";
+import List from "./routes/List";
+import Population from "./routes/Population";
+import Gdp from "./routes/Gdp";
+
+export default function App() {
   return (
     <>
-      <header className="h-screen flex justify-center items-center bg-red-700">
-        <h1>Hello</h1>
+      <header className="w-full p-6 text-white bg-slate-700">
+        <h1 className="mb-4 text-xl text-center sm:text-3xl sm:font-bold">
+          HW3
+        </h1>
+        <NavBar />
       </header>
+      <Routes>
+        <Route path="/" element={<Home />} />
+        <Route path="/list" element={<List />} />
+        <Route path="/population" element={<Population />} />
+        <Route path="/gdp" element={<Gdp />} />
+      </Routes>
```

Comment:
I think this is a way to do the route, and that's fine. I saw this online a lot too, but wound up using the same format we did in class with a link to the component rather than a route.

Reply:
I agree.  I've seen this done pretty commonly as well.  It's nice that you used the App.jsx file!  I just deleted it because it seemed to be redundant.  Couldn't figure out how to purpose each file in the file structure.